test/tsconfig.json

```diff
@@ -7,10 +7,12 @@
 		"removeComments": false,
 		"preserveConstEnums": true,
 		"sourceMap": false,
-		"experimentalDecorators": true
+		"experimentalDecorators": true,
+    "typeRoots": [
```

Comment:
Spacing mismatch, according to `.editorconfig` json files should (indeed) have 2 spaces, can you reformat the other lines?